src/_premake_init.lua

```diff
@@ -590,6 +590,12 @@
 		kind = "string",
 	}
 
+	api.register {
+		name = "csversion",
```

Comment:
These are supposed to be in alphabetical order, I think `dotnetframework` was renamed from `framework` which is why it's here and not up with the other `d*` APIs.